### ISSUE TITLE
add illustrations by allison horst

### ISSUE THREAD

**Issue by lnnrtwttkhn** [lnnrtwttkhn/version-control-book#97]
https://allisonhorst.com/git-github

don't forget to credit license / author!

**Comment by konradpa** [comment]
Could be added to: "Remotes: Intro" chapter

**Comment by lnnrtwttkhn** [comment]
Yes, good idea. Could you do a PR in the 'images' repo and we later reference it in the book.
